### PR TITLE
New: tag prefix processor

### PR DESCRIPTION
**Pull request by jping0929**

### CODE REVIEW

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1062",
    "commit-sha": "51c6102c28b1c2e0b16dbe825d48d549510fd618",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0310da17b4e445d4438b63a676238df160d2574e",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1062",
    "commit-sha": "889de4909810543ff100092945236520b97b10eb",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0310da17b4e445d4438b63a676238df160d2574e",
    "pipeline": "master"
  }
]
```

File: specs/tagprefix.spec [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Model
+model:
+  rest_name: tagprefix
+  resource_name: tagprefixes
+  entity_name: TagPrefix
+  package: squall
+  group: core/namespace
+  description: Returns the tagprefixes of the specified namespace.
```
Comment (ericrpowers) [inline specs/tagprefix.spec:8]: tagprefixes should be "tag prefixes"

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1062",
    "commit-sha": "889de4909810543ff100092945236520b97b10eb",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Review by ericrpowers — APPROVED** [review]

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1062",
    "commit-sha": "889de4909810543ff100092945236520b97b10eb",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "abaf7bed5a9262f9bf3aed3c357dac98fbbfe8db",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "abaf7bed5a9262f9bf3aed3c357dac98fbbfe8db",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "abaf7bed5a9262f9bf3aed3c357dac98fbbfe8db",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "1624737d3ce73db02d3b47ceaeb9b2ca66ca9ea9",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "8b038412c22d2b092211db1204a81c221448fdf6",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "8b038412c22d2b092211db1204a81c221448fdf6",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "618",
    "commit-sha": "0891ab589382515d33cdb461b2ad55d13521b853",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "1065",
    "commit-sha": "57be71a81e4d8af07e2be913324589df6288a22f",
    "pipeline": "master"
  }
]
```

**Pull request merged**